### DISCUSSION TITLE
Animation looping in AR in iOS (Safari)

### DISCUSSION THREAD

**Discussion by kewalkishang** [google/model-viewer#3366] [Q&A] [answered]
I have an experience where I am using a button to play an animation once. This works fine in android. While in iOS it works fine in the 3D view but when I go into AR the object keeps looping the animation. Am I missing anything ?  
https://maple-summer-branch.glitch.me/
Also annotations needs iOS to start supporting webxr right ?

**Comment by elalish** [comment, marked as answer]
Yes, and any custom browser logic, like your HTML button and JS action also require the browser to work, and thus require WebXR for AR. Quick Look is a native app that only takes a USDZ as input; the website is entirely left behind. Please file a bug on WebKit or otherwise ask Apple to prioritize this.
